### ISSUE TITLE
SKILL.tsv discussion and coordination.

### ISSUE THREAD

**Issue by Nagoda** [Treeofsavior/EnglishTranslation#672]
With the English test starting tommorow, I figured it would be good to have some place to discuss the recently added SKILL.tsv file. In particular, since stuff like skill descriptions are really formulaic and generally follow a template, it's important for us to coordinate and make sure we're following similar guidelines as best we can. The UI file probably could use a similar discussion thread for similar reasons, but I haven't had much time to look at it yet.

I'm probably not going to make any pull requests myself until after the beta period ends, or at least a few days into it, but I have skimmed over the skill file and spotted some things. In particular there are a ton of descriptions with awkward tenses and prepositions that will need some finessing.

One of the big obvious problems I spotted was that there is a description "template" that is used uniformly in Korean to describe how skill damage increases with attribute level, and it's translated inconsistently a number of different ways in English (none of them sound good)

The Korean looks something like:

적에게 입히는 [Skill Name Here]의 대미지가 특성 레벨당 1% 증가합니다.

but it's translated in English variably as:

"Damages dealt on an enemy with [Skill Name] will increase by 1% per Attribute level."
"The amount of [Skill Name] damages incurred on an enemy increases by 1% per Attribute level."
"Damages that will be incurred on an enemy by [Skill Name] will be increased by 1% per Attribute level."
"Damages of [Fire Wall] that will be dealt on an enemy will increase by 1% per Attribute level."

It's different for almost every class. It seems pretty clear that we need to decide on a single template that all of these descriptions follow.

These lines should probably all be uniformly changed to something more like:

-"Damage dealt by [Skill Name] increases by 1% per Attribute level."
or
-"[Skill Name]'s damage increases by 1% per Attribute level."

In general, there are also a lot of descriptions written using future tense, particularly for what seems to be "perks" or "talents" that modify/improve existing skills, i.e.:

-"SP consumption of [Skill Name] will be reduced."

I think the future tense is pretty odd sounding in this context and should be avoided in all of these descriptions. There is no real grammatical rule about this, but other games with RPG systems are not usually written this way. The word "will" probably shouldn't appear anywhere. I would change these lines to something more like:

-"SP consumption of [Skill Name] reduced."
or
-"Reduces SP consumption of [Skill Name]."

I would also broadly add that "damages" as a noun should not be used anywhere in the game. In English, "damages" (noun) is most closely associated with a monetary award resulting from a legal dispute. "Damage" is definitely the preferred noun.

Feel free to add your thoughts, or bring up other issues within the Skill.tsv file to get opinions.

**Comment by wlghksdl2** [comment]
I agree with you. Templates and clear guidelines for editing must be established now that the translations seem to be close to being finished. I hope imcgames will provide one before these editorial conflicts get out of hand. 

As for the Attributes, I believe the reader should be able to grasp the nature of the attribute from the first few words. 
For example, 
"Damage dealt by [Skill Name] increases by 1% per Attribute level."
The first two words provide a summary. From a quick scan, the player knows this attribute affects the damage portion of the said skill. I believe this format is better than the second example you provided.

I also agree future tense is redundant. Further, I believe present tense provides the best clarity for the skills and items being described.

If possible, we should try to begin every descriptions of skills with a verb. Again, the first word should provide a clear summary of the item/skill being described as I have stated earlier. Even more so in-game, since the name of the attribute/item and the visible icons aid the players in understanding its nature. As seen in your last example, "Reduces SP consumption of [Skill Name]," the player immediately knows the Attribute is related to SP cost of the skill without having to read the entire description.

**Comment by Ihysoal** [comment]
Better to use "cost" rather than "consumption", it's shorter and one of the top 1000 used English words.

As for the skill descriptions, what is the context? Where is it placed? Do we need [Skill Name] for some reason? Why is it attribute level instead of attribute point?

Do you see these when you hover over the attribute in question in a tooltip? Then you can leave attribute out all together and just say point (or level).

Increases Fire Wall damage by 1% per point.

I mean, in reality it doesn't really matter which words come first, you basically need to read the whole sentence to understand what it actually does. The only irrelevant part (and thus, last) is "per point", since it's pretty much implied.

Whether you say:
Increases Fire Wall damage by 1% per point. (Sounds right)
Damage of Fire Wall increases by 1% per point. (Extra "of", sounds off)
Fire Wall's damage increases by 1% per point. (Sounds right, but beware skills ending in 's' here.)

In the end you need to know 3 things as fast as possible:
- What skill,
- Which of the skill's attributes is affected,
- How is it affected?

In what order is rather irrelevant.

Of course, if the tooltip is somewhere else, my whole suggestion might not apply. Hence, context.

**Comment by Nagoda** [comment]
Agreed about "cost" vs "consumption"

One of the reasons I'm only bringing this up in discussion and am not going to make any pull requests until after I can spend some time with the game is that I don't really understand how the skill and progression systems work, or how this text is presented in the game. 

We'll have a much better sense of how it should look in a few days, and we can get more specific then. Mostly with this opening post I just wanted to make clear the need for consistency, and to start thinking about broad guidelines.

**Comment by wlghksdl2** [comment]
G1, you missed the point of the post entirely.

You are right that vital information must be presented as fast as possible, but the order IS relevant to establish, as Nagoda suggests, a template and guidelines for the editors to follow to improve unity and coherence throughout different skill and item descriptions in the game

**Comment by Ihysoal** [comment]
No, I understood your post perfectly, you want consistency. Most dedicated and wannabe-dedicated editors want that. What I was making a comment about was the order you seemed to focus on. I figured its consistency across skills was kind of implied with that.

You cannot give a clear summary in one word for skill descriptions\* and "damage" is not a verb to start a sentence with as you proposed later on.

The order is rather irrelevant, only that it is consistent across all skill descriptions. (Even that is debatable. I can't read Korean, so I don't know what the consistency is there. But when I look at, for example, the ambiguous ellipsis usage in the Korean text, that's a rather big inconsistency itself.)

\* Technically, you could make the skill an icon and then say +1% damage or -1% damage and that would give a 1 word summary of what changes, and it would be easy to understand too. Maybe they'll do that eventually?

**Comment by wlghksdl2** [comment]
To say the order, that is sentence structure, is irrelevant seems nonsensical. Through sentence structure we can emphasize certain parts as well as deliver its meaning more quickly.

I gave an example of my suggestion to try to begin as many descriptions using verbs. Rather than -"SP consumption of [Skill Name] reduced," I suggested -"Reduces SP consumption of [Skill Name]" was the better choice. Obviously not all descriptions can start like so, but I suggest, as much as possible, to format the descriptions like the following:

[How it is affected], [Which of the skill's attributes is affected], and [What skill] and lastly [Necessary extra information]. 

The players will be most interested in how their skills are affected; therefore, giving that information first in the descriptions seems most reasonable. Further, when there is a long list of attributes the player can choose and purchase from, this format should prove less burdensome to read.

**Comment by Ihysoal** [comment]
> The players will be most interested in how their skills are affected

It seems you just don't get it, until you read the final detail you won't know what skill is affected in your example. Ergo, the order doesn't really matter.

Edit: http://puu.sh/joR9Z/bc69ba77dd.jpg
http://puu.sh/joQQf/5cdea3a4ea.jpg

This seems to be the context of these descriptions.

Also. Perhaps imcgames can clarify how far they mind that things deviate from the original translation. Heck, you know where the tooltip is coming from and the title already describes the ability being affected. Do we need to state things like [Skill Name] double?

**Comment by g1real** [comment]
http://puu.sh/joQ4e/255771a963.jpg

This seems to be the context of these descriptions. I would do something like

>Enhanced Heal
>[Heal] does 1% extra damage per level.

>Additional Heal
>Gain a 2% chance to cast another [Heal] per level.

Perhaps imcgames can clarify how far they mind that things deviate from the original translation. Heck, you know where the tooltip is coming from and the title already describes the ability being affected. Do we need to state things like [Skill Name] and Attribute double?

**Comment by wlghksdl2** [comment]
I beg to differ.

I will use one of the examples you provided.

Cure: Enhance
Damage dealt on an enemy with [Cure] will increase by 1% per Attribute Level.

If this were to be reworded

Cure: Enhance
Increases damage dealt on an enemy with [Cure] by 1% per Attribute Level.

The player only needs to read the first two words to understand HOW Cure:Enhance changes Cure. If the player is interested in dealing more damage, he can read it whole, if not, the player doesn't even have to read the rest of the description, thus saving time.

I have already raised an issue with redundancy in item and skill descriptions. Since much of the information can be easily inferred by the name of the skill/item and information available in-game, we can chop away a lot of the deadwood.

For example, 
Cure:Enhance
Increases damage dealt by 1% per Attribute Level (or per point as you suggested earlier)

**Comment by kyuur** [comment]
Several skills have a section called 'Range Attack Ratio'. After some testing, I believe this refers to the number of enemies that can be hit at once. It should be changed to the same as:

ETC_20150317_002785 AoE Attack Ratio

Which I believe reflects the same thing for regular attacks, or a new term such as 'Max Targets Hit' or similar.

Not sure if worth raising a brand new issue or if this should be contained to 'discussion' (maybe will get confusing with too many topics at once?)

**Comment by ThiefZero** [comment]
> Cure: Enhance
> Damage dealt on an enemy with [Cure] will increase by 1% per Attribute Level.

vs

> Cure: Enhance
> Increases damage dealt on an enemy with [Cure] by 1% per Attribute Level.

The last one is much easier to read and understand (though ideally, I think _"Increases damage dealt with [Cure] by 1% per Attribute Level"_ would be better, as damage would only be dealt to enemies). I agree with @wlghksdl2, we should prioritize skills using 

> [How it is affected], [Which of the skill's attributes is affected], and [What skill] and lastly [Necessary extra information]

as a guideline.

**Comment by Nagoda** [comment]
I agree that for very simple lines like "increases damage dealt" we can probably remove the "to the enemy" clause when it is implicit and unnecessary.

> [How it is affected], [Which of the skill's attributes is affected], and [What skill] and lastly [Necessary extra information]

This might be too broad for us to reliably apply, but if we can lay out a clear rule like

> 적에게 입히는 [Skill Name]의 대미지가 특성 레벨당 1% 증가합니다.

should always be translated as

> Increase the damage of [Skill Name] by 1% per Attribute level.

then that's much more specific and we can agree or disagree (and offer concrete alternate suggestions rather than quibbling over ambiguous rules) very easily. I don't think it's going to be possible for us to come up with a single universal template for all of these descriptions, but what we can do is identify when there is a repeatedly used Korean template, and come up with an appropriate corresponding English template, such as the line I quoted above.

**Comment by wlghksdl2** [comment]
On top of providing guidelines and templates for editors to follow, I believe imcgames needs to appoint head editor(s) whose edits will be considered final. Then other editors can rally around them, observe and copy their editorial style thus improving consistency throughout.

**Comment by Nagoda** [comment]
The text frequently describes enemies as "falling under" or "into" a status affliction, which sounds pretty awkward I think. Examples:

>   Enemies attacked with [Ice Blast] falls under [Freeze] by chance of 10% per attribute level.
> Enemy falls under [Stun] for 1 second by 10% chance per Attribute level when hit with [Psychic Pressure].
> Enemies attacked with [Effigy] falls under [Blind] state at chance of 5% per Attribute level.

My first thought was maybe "suffer" was a better verb, but given most of the negative status effects are written as nouns it ends up sounding a bit goofy ("Enemies suffer [Blind]"). I think "inflict" works better for our purpose here. An example cleanup might be:

> [Ice Blast] has a 10% chance per Attribute level of inflicting [Freeze].
> [Psychic Pressure] has a 10% chance per Attribute level of inflicting [Stun] for 1 second.
> [Effigy] has a 5% chance per Attribute level of inflicting [Blind].

We can remove the reference to the enemy because it's implicit. There are a lot of abilities that seem to have procs tied to usage of a parent skill, and the language they use to communicate this is pretty inconsistant, so this is also one possible template we can use for skills that have an x% chance of inflicting a debuff.

**Comment by wlghksdl2** [comment]
That template seems very straightforward for Attributes that give procs.
How about turning [Status] into an active verb instead?
[Ice Blast] has a 10% chance per Attribute level to freeze enemies.
[Psychic Pressure] has a 10% chance per Attribute level to stun for 1 second.
[Effigy] has a 5% chance per Attribute level to blind enemies.

**Comment by wlghksdl2** [comment]
Alternatively, we can rid of Skill names for concision. 

In-game it will look like this:

Ice Blast: Enhance (or whatever the name of the Attribute)
10% chance per Attribute level to freeze enemies

Psychic Pressure: Enhance
10% chance per Attribute level to stun for 1 second

Effigy: Enhance
5% chance per Attribute level to blind enemies.

So for Attributes that give proc/debuff effects, template can be
x% Chance per Attribute level to [Status] enemies [extra information such as duration]

or 

[Skill name] has a x% chance per Attribute level to [Status] enemies [extra information such as duration]
If the skill being enhanced by the Attribute is not implied by Attribute name

**Comment by Nagoda** [comment]
It would be much more elegant if we could simply treat the status debuffs as verbs in a sentence, and many of them would work very well like that, but some don't really function as verbs, or would need to be conjugated to work. Just skimming the tsv file, I see Bleeding, Defense Weakening, Armor Break, Flame, Confusion, and more.

Treating the debuffs all as nouns I think is just more flexible. We can refer to the "status effect" itself as the noun, that way it doesn't matter what the name of the debuff is, it will always work.

I think keeping the name of the skill in the description is a good thing for clarity. It will help new players understand that these attributes modify existing skills by adding new effects to them, rather than having to intuit this from the name of the attribute alone. I don't think we need to be quite that ruthless with cutting down the length of these descriptions.

**Comment by wlghksdl2** [comment]
Sounds very convincing to me.

I dare say these templates are very reasonable to be put to use now. Let's see how they hold up and bring up any issues here.

I will try my hand editing Skil.tsv Lines 1-300 and see how it goes.
Nevermind, seems like Attributes start around line 1965, HP Potion Booster.

**Comment by wlghksdl2** [comment]
Swash Buckling: Max. HP
Increases Max. HP by 5% per Attribute Level while [Swash Buckling] is active.

Or

While [Swash Buckling] is active, increases Max. HP by 5% per Attribute Level

I prefer the former. What do you think?

**Comment by wlghksdl2** [comment]
I have standardized Attribute Descriptions from 1967-2111. https://github.com/Treeofsavior/EnglishTranslation/pull/691
If anyone else wants to continue from there, please let us know here. If not, I will continue sometime tomorrow.

**Comment by ThiefZero** [comment]
What about:

> Increases chance to apply [Bleed] with [Backstab] by 1% per Attribute Level.

and for @g1real [here](https://github.com/Treeofsavior/EnglishTranslation/pull/691#issuecomment-127892598)

> Increases chance to double cast with [Heal] by 1% per Attribute Level.

**Comment by wlghksdl2** [comment]
That's quite an elegant suggestion. Thanks you two.

[Increase/Decrease] chance to apply [Status Effect] with [Skill] by x% per Attribute Level.

Similarly,

[Increase/Decrease] Chance to [Proc Effect] with [Skill] by x% per Attribute Level.

I will edit my pull request and see how that goes.

**Comment by wlghksdl2** [comment]
Upon closer examination,

[Spear Blow] has a 5% chance per Attribute Level to inflict Bleed (or to apply Bleed)
Increases chance to apply Bleed with [Spear Blow] by 5% per Attribute Level.

the former is shorter, but the latter is more direct at delivering exactly what the attribute does.

I am leaning towards the latter. What do you guys think?

**Comment by ThiefZero** [comment]
Latter is easier to read. We should probably also tag @imcgames to let them know about this discussion, and if they're alright with final choice. Maybe they want to decide on the final matter.

**Comment by MrLiar** [comment]
hey guys i am currently editing some of Skill.tvs, would it be better to use enemy or target ?
